backend/api/Areas/Leases/Models/Lease/SecurityDepositModel.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+
+namespace Pims.Api.Areas.Lease.Models.Lease
+{
+    public class SecurityDepositModel
+    {
+        /// <summary>
+        /// get/set - Primary key to identify Security Deposit.
+        /// </summary>
+        public long Id { get; set; }
+
+        /// <summary>
+        /// get/set - The concurrency row version.
+        /// </summary>
+        /// <value></value>
+        public long RowVersion { get; set; }
+
+        /// <summary>
+        /// get/set - Foreign key to the security deposit holder.
+        /// </summary>
+        public string SecurityDepositHolderTypeId { get; set; }
+
+        /// <summary>
+        /// get/set - Security deposit holder.
+        /// </summary>
+        public string SecurityDepositHolderType { get; set; }
```

Comment:
instead of passing the type as a string, it could be an object. that way the id (type code) and description are encapsulated

Reply:
I'm following the same pattern as other models that were written before me. Let's discuss best practice moving forward and create a tech debt ticket. what do you think?

Reply:
I think it works either way. The only problem I see is that the type information is not encapsulated, leaving the naming to show the relationship. Non blocking either way

Reply:
Agree and happy to switch to an object but would prefer to merge the PR and then agree on all the instances where we are using this pattern moving forward.